test/support/test_helpers.ex

```diff
@@ -1,39 +1,65 @@
 defmodule RemoteRetro.TestHelpers do
   use Wallaby.DSL
-  alias RemoteRetro.{Repo, User, Vote}
+  alias RemoteRetro.{Repo, User, Vote, Idea, Participation}
 
   @mock_user Application.get_env(:remote_retro, :mock_user)
 
-  def persist_idea_for_retro(context) do
-    %{idea: idea, retro: retro, user: user} = context
-
-    idea = if idea.category == "action-item" do
-            Map.merge(idea, %{retro_id: retro.id, user_id: user.id, assignee_id: user.id})
-            |>Repo.insert!
-          else
-            Map.merge(idea, %{retro_id: retro.id, user_id: user.id})
-            |>Repo.insert!
-          end
-
-    Map.put(context, :idea, idea)
-  end
-
-  def use_all_votes(%{user: user, idea: idea} = context) do
+  def use_all_votes(%{test_user: user, idea: idea} = context) do
     now = DateTime.utc_now
     vote = [user_id: user.id, idea_id: idea.id, inserted_at: now, updated_at: now]
     Repo.insert_all(Vote, [vote, vote, vote, vote, vote])
     context
   end
 
-  def persist_user_for_retro(context) do
-    context = Map.merge(%{user: @mock_user}, context)
-    %{user: user} = context
+  defp user_name_atom(name) do
+    String.replace(name, ~r/ +/, "") |> Macro.underscore |> String.to_atom
+  end
+
+  defp user_map(users) do
```

Comment:
I think we can do away with `user_name_atom` helper, as `Map`s can house string keys. This avoids the need for the helper, as well as the transformations done when building the map and accessing it. Separately, I think we should use the users `email` as the key in this map, rather than the user's name, as [email is unique](https://github.com/stride-nyc/remote_retro/blob/master/web/models/user.ex#L36).

Reply:
i thought about that but i was afraid that would make the call sites in `retro_channel_test.exs` sort of wierd that you'd have to know the email address of your mock user to use it in the tests.  

also, i liked having the atom...

if anything i was thinking that we should refactor mock_user to be test_user so it's consistent with the name - or else change mock_user's name from test_user to mock_user (same difference really).  

I don't feel particularly strongly so up to you!

Reply:
> you'd have to know the email address of your mock user to use it in the tests

Good point. What if we changed the test users' names and emails to be something semantic, like `@test_user_one`, where the email is `testuser@one.com`, and `@test_user_two`, where the email is `testuser@two.com`. We could do the same thing with the names, as I think that would make things much clearer for all folks who ending up reading/writing tests?

Reply:
sounds good.  i'll make that change.

Reply:
made the change but left the names "Test User" and "Other User" because i felt it made it easier to read the error messages on failing tests.  I think it clarifies things, though.  This should be good to merge now.  Lemme know!